modules/command/dh/dh-pay-out-command.js

```diff
@@ -34,6 +31,10 @@ class DhPayOutCommand extends Command {
             blockchain_id,
         } = command.data;
 
+        if (!command.data.retryNumber) {
+            command.data.retryNumber = 1;
```

Comment:
Is this for the existing commands?

Reply:
Yes, for existing and also for new commands